pika/adapters/twisted_connection.py

```diff
@@ -405,6 +408,32 @@ def _send_frame(self, frame_value):
         self.bytes_sent += len(marshaled_frame)
         self.frames_sent += 1
         self.transport.write(marshaled_frame)
+        
+# fixed here
+    def _send_message(self, channel_number, method_frame, content=None):
+        """Send the message directly, bypassing the single _send_frame
+        invocation by directly appending to the output buffer and flushing
+        within a lock.
+
+        :param int channel_number: The channel number for the frame
+        :param pika.object.Method method_frame: The method frame to send
+        :param tuple content: If set, is a content frame, is tuple of
+                              properties and body.
+
+        """
+        length = len(content[1])    
+        self._send_frame(frame.Method(channel_number, method_frame))
+        self._send_frame(frame.Header(channel_number, length, content[0]))
+        
+        if content[1]:
+            import math
```

Comment:
Why import math here? The one at the top of the file is the right and only place for it.